solana/bridge_token_factory/programs/bridge_token_factory/src/constants.rs

```diff
@@ -0,0 +1,25 @@
+use anchor_lang::prelude::*;
+
+#[constant]
+pub const CONFIG_SEED: &[u8] = b"config";
+
+#[constant]
+pub const AUTHORITY_SEED: &[u8] = b"authority";
+
+#[constant]
+pub const VAULT_SEED: &[u8] = b"vault";
+
+#[constant]
+pub const USED_NONCES_SEED: &[u8] = b"used_nonces";
+
+#[constant]
+pub const WRAPPED_MINT_SEED: &[u8] = b"wrapped_mint";
+
+#[constant]
+pub const USED_NONCES_PER_ACCOUNT: u32 = 1024;
+
+#[constant]
+pub const USED_NONCES_ACCOUNT_SIZE: u32 = 8 + (USED_NONCES_PER_ACCOUNT + 7) / 8;
```

Comment:
Can you please add a comment where the extra +8 and +7 came from.

Reply:
Anchor discriminator